data.md

```diff
@@ -78,6 +18,18 @@ module WASM-DATA
     imports BYTES
 ```
 
+### WASM Token Sorts
```

Comment:
I'm a little confused by this: The `syntax` declarations already had `token` attributes, so why was this change necessary? This almost seems like a frontend bug to me.

Reply:
The module used for verification is `KWASM-LEMMAS`, which doesn't have `WASM-TOKEN-SYNTAX` in its import chain. We only ever import `WASM-TOKEN-SYNTAX` when trying to parse programs.

Reply:
That's strange, because I was also getting errors due to the missing `token` attribute when trying to run concrete programs.

Reply:
I believe this should be resolved now.